test/employees-service.test.ts

```diff
@@ -0,0 +1,105 @@
+import {
+    TestBed, fakeAsync, tick
+} from '@angular/core/testing';
+import { Component } from '@angular/core';
+import { MockModule } from './mock.module';
+import { EmployeesService, createEmployee, employeeTitles } from '../src/employees.service';
+import { HttpClientModule } from '@angular/common/http';
+
+const fields = ['managerId', 'id', 'name', 'firstName', 'lastName', 'title', 'active', 'hireDate', 'phone'];
+const testHier = (data) => {
+        expect(data[0].reports[0].reports[1].reports[0]).toEqual(akeemEmployee);
+};
+const akeemEmployee = {"active": true, "firstName": "Akeem", "hireDate": new Date("2017-01-15"), "id": 60, "lastName": "Carr", "name": "Akeem Carr", "phone": 7381362814, "title": "Junior Software Developer"};
+
+describe('employees', () => {
+    @Component({
+        template: ''
+    })
+    class TestComponent {
+        constructor(public service: EmployeesService) {}
+    }
+    let service;
+    beforeEach(() => {
+        TestBed.configureTestingModule({
+            declarations: [TestComponent],
+            imports: [MockModule,  HttpClientModule],
+            providers: [EmployeesService]
+        });
+        const template: string = '<kendo-component></kendo-component>';
+        TestBed.overrideComponent(TestComponent, {set: { template }});
+        const fixture: any = TestBed.createComponent(TestComponent);
+        service = fixture.debugElement.injector.get(EmployeesService);
+    });
+
+
+    it('should inject service EmployeesService', () => {
+        expect(service).toBeTruthy();
```

Comment:
Could be anything with `toBeTruthy`. Better assert on the specific type using `toBeInstanceOf`.

Reply:
I reached some TS limitations on this initially , will try again